### ISSUE TITLE
Design GUI for steering wheel

### ISSUE THREAD

**Issue by i315315** [amosproj/amos2022ws02-automotive-test-app#37]
## User story
1.	As a User
2.	I want to be able to enter an area in the test app where I can see a multifunctional steering wheel with a selection of buttons on it
3.	So that I can get an overview of which button simulations are available. 

With this User story, the graphical user interface of the steering wheel is to be designed. For inspiration see the Notes section below.

The following buttons are needed (the list might be extended with upcoming requirements):
* SkipForward
* SeekForward (natively available from Android)
* Play/Pause
* Voicecontrol
* Phone (Event „HeadsetHook()")

## Acceptance criteria
* The graphical user interface of the steering wheel is designed.

## Notes
Inspiration for GUI design: 
* https://www.thedrive.com/content-b/message-editor/1623169499432-m163657-2.jpg?quality=60
* https://www.bedienungsanleitu.ng/audi/a4-2021/anleitung?p=15
* https://www.bedienungsanleitu.ng/audi/a4-2021/anleitung?p=31

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
